widget/embedded/src/pages/Home.tsx

```diff
@@ -115,13 +133,31 @@ export function Home() {
 
   useEffect(() => {
     resetQuoteWallets();
+    setRefetchQuote(true);
   }, []);
 
   const percentageChange =
     !inputUsdValue || !outputUsdValue || !outputUsdValue.gt(0)
       ? null
       : getPriceImpact(inputUsdValue.toString(), outputUsdValue.toString());
 
+  const selectQuoteByTag = (tag: {
+    label: string;
+    value: TagValue | string;
```

Comment:
The union type consisting of strings and string type will be reduced to just string

Reply:
fixed